understanding/21/pointer-gestures.html

```diff
@@ -32,11 +48,10 @@ <h3>Benefits</h3>
 		<section id="examples">
 			<h2>Examples</h2>
 				<ul>
-        <li><p>A web site includes a map view that supports both the pinch gesture to zoom into the map content and drag gestures to move the visible area. User interface controls offer the operation via [+] and [-] buttons to zoom in and out, and arrow buttons to pan stepwise in all directions.</p></li>
+        <li><p>A web site includes a map view that supports the pinch gesture to zoom into the map content. User interface controls offer the operation via [+] and [-] buttons to zoom in and out.</p></li>
```

Comment:
We can probably remove " via [+] and [-]"

Reply:
Just think about how that will read with a screen reader...

Reply:
done.